version-compatibility/forkless-upgrade/src/backward_compatibility.rs

```diff
@@ -47,15 +47,14 @@ async fn latest_binary_is_backward_compatible_and_can_load_testnet_config() {
 }
 
 #[tokio::test]
-async fn latest_binary_is_backward_compatible_and_follows_blocks_created_by_genesis_binary(
-) {
+async fn latest_binary_is_backward_compatible_and_follows_blocks_created_by_v36_binary() {
```

Comment:
It still would be nice to have one more backward compatibility tests that produces blocks by `fuel-core 0.26.0`

Reply:
Genesis test is restored in https://github.com/FuelLabs/fuel-core/pull/2271/commits/13f26532454896633cac084c936d265c100049fc